### PR TITLE
Update README.md

### LINKED ISSUE #1401: There are typos in README.md
**Actual Behaviour**
There are typos in README.md

**Expected Behaviour**
Typos should be removed.

**Would you like to work on the issue?**
Yes, I would like to work on it.

Please let us know if you can work on it or the issue should be assigned to someone else.

### PR DESCRIPTION
**Pull request by jishnujayakumar**
Fix #1401 

Typos in README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,12 +31,13 @@ Phimp.me is a Photo App for Android that aims to replace proprietary photo appli
   
 ---
 
+Do you have many different apps in your Android Phone for the purpose of clicking, editing and sharing images? Hey guess what, now you don't need all of those because `Phimpme` is here, with all the features packaged into one. So go and clean your phone memory to save your space for other useful purposes.
```
Comment (mayank8318) [inline README.md]: Name of the app is **Phimp.me**. Please correct this also if it is wrong.

File: README.md [hunk 2]
```diff
@@ -31,12 +31,13 @@ Phimp.me is a Photo App for Android that aims to replace proprietary photo appli
...
+
 ## Features
-Do you also have many different apps in your Android Phone for image click, editing and sharing purpose? Now clean your phone memory. Phimpme is here, with all the features packaged into one.
 
 * Click beautiful image from Phimpme app. Use various advance scene modes, and variety of While balance modes.
 * You can use your voice actions to invoke Camera, Just say "Ok Google click a picture". You can also select front and rear camera based on voice.
-* Browse the local gallery inside the app with folder and all photos mode. Copy, move to images, add description the images.
+* Browse the local gallery inside the app with folder and all photos mode. Copy, move and add description to the images.
 * Edit images with various advance filters, optimized performace using OpenCV library.
```
Comment (mayank8318) [inline README.md]: *various advanced/cool filters.

Comment (mayank8318) [reply README.md]: Whichever sounds exciting

Comment (jishnujayakumar) [reply README.md]: Done #14

Comment (jishnujayakumar) [reply README.md]: @sridharjajoo #1402

File: README.md (hunk 1 above)
Comment (sridharjajoo) [reply README.md]: I agree with @mayank8318 . @jishnujayakumar Kindly do the required changes.

**Pull request closed without merging**